### PR TITLE
Add our path resolution policy to developing

### PR DESCRIPTION
**Pull request by Miauwkeru**
(DIS-2622)

### CODE REVIEW

File: docs/source/contributing/developing.rst [hunk 1]
```diff
@@ -19,6 +19,39 @@ These pages are organized as follows:
...
+
+The ``--resolve`` adds, as the name, the resolved path to the record and transforms the information above to:
+
+.. code-block::
+
+    some_name=%SystemRoot%\Temp\file
+    some_name_resolved=C:\Windows\Temp\file
+
+
+While the ``--hash`` flag adds extends the ``--resolve`` information and includes the ``digest`` to it.
```
Comment (Schamper) [inline docs/source/contributing/developing.rst:46]:
```suggestion
The ``--hash`` flag extends the ``--resolve`` information and includes a ``_digest`` suffixed field as well, which contains the file digest triple (if available):
```

**Review by Schamper — CHANGES_REQUESTED** [review]

File: docs/source/contributing/developing.rst (hunk 1 above)
Comment (Schamper) [reply docs/source/contributing/developing.rst:46]: The `if available` was a reference to if the file exists, but I've reworded it.

File: docs/source/contributing/developing.rst [hunk 2]
```diff
@@ -19,6 +19,39 @@ These pages are organized as follows:
 - :doc:`Tooling </contributing/tooling>`: information regarding the available tooling for building and testing code and documentation.
 
 
+Path resolution policy
+----------------------
+
+Starting from Dissect version 3.11, we decided that new plugins will only provide the original parsed ``path`` value in its records.
```
Comment (pyrco) [inline docs/source/contributing/developing.rst:25]:
```suggestion
Starting from Dissect version 3.11, the policy is for plugins to only provide the original parsed ``path`` value in returned records.
```

File: docs/source/contributing/developing.rst [hunk 3]
```diff
@@ -19,6 +19,39 @@ These pages are organized as follows:
...
+We recommend external contributors to follow the same policy.
+This keeps the information inside the records as close to the truth as possible.
```
Comment (pyrco) [inline docs/source/contributing/developing.rst:27]:
```suggestion
We recommend external contributors to follow the same policy,
this keeps the path information in the records as close to the truth as possible.
```

File: docs/source/contributing/developing.rst [hunk 4]
```diff
@@ -19,6 +19,39 @@ These pages are organized as follows:
...
+
+A resolved path can still be accessed by using the ``--resolve`` or ``--hash`` flags in :doc:`target-query </tools/target-query>`.
```
Comment (pyrco) [inline docs/source/contributing/developing.rst:29]:
```suggestion
Resolved paths can be accessed by using the ``--resolve`` or ``--hash`` flags in :doc:`target-query </tools/target-query>`.
```

**Review by pyrco — CHANGES_REQUESTED** [review]

**Review by pyrco — DISMISSED** [review]

**Review by Schamper — APPROVED** [review]

**Pull request merged**